cli/cmd/compose/convert.go

```diff
@@ -47,6 +50,10 @@ func convertCommand(p *projectOptions) *cobra.Command {
 	flags := convertCmd.Flags()
 	flags.StringVar(&opts.Format, "format", "yaml", "Format the output. Values: [yaml | json]")
 
+	// add flags for hidden backends
+	for _, f := range addFlagsFuncs {
```

Comment:
I don't see any reason this is restricted to kube backend. 
ECS and local can also use this to output to a file